web/templates/blocks/sap_systems_table.html.tmpl

```diff
@@ -77,15 +77,22 @@
                                     {{- else }}
                                         <td></td>
                                     {{- end }}
-                                    <td><a href="/clusters/{{ .ClusterID }}">{{ .ClusterName }}</a></td>
+                                    <td>
+                                    {{- if ne .ClusterType "Unknown" }}
```

Comment:
Good catch! It wouldn't be safer to check if the cluster is of HANA type?

Reply:
This is actually future proofer, since we do not have to change this condition when we will introduce netweaver cluster detail pages